shellcheck/shellcheck.yml.yaml

```diff
@@ -0,0 +1,15 @@
+files:
+  - repo: console-login-helper-messages
+    path: .github/workflows/shellcheck.yml
+    vars:
+      branches:
+        - main
+
+  - repo: fedora-coreos-config
+    path: .github/workflows/shellcheck.yml
+    vars:
+      branches:
+        - branched
+        - next-devel
+        - rawhide
+        - testing-devel
```

Comment:
Thinking more on this, I think in theory only `testing-devel` is necessary. Of [the few things](https://github.com/coreos/fedora-coreos-releng-automation/blob/26fc90ce012fa5eb64ed0b082e4b195a5c67b563/config-bot/config.toml#L38-L42) that aren't synced across to the other branches, none of them are shell scripts (except shell scripts embedded in `manifest.yaml` as postprocess scripts, which we don't check yet but would be cool to add).

Obviously it should still work to run CI on those branches, but I'm wondering if it makes sense to do it on PRs against those other branches if fixes for CI failures would actually need to land in `testing-devel` anyway (though that shouldn't even happen for it to have landed in `testing-devel` in the first place).